### ISSUE TITLE
Tags Configuration Documenation

### ISSUE THREAD

**Issue by eavestn** [DataDog/dd-trace-js#111]
**Description** 

I would like to take advantage of the `tags` listed in the _Integrations_ section of the [documentation](https://datadog.github.io/dd-trace-js/#express__anchor). 

**Issue**

_How_ to configure the tags is not explicitly addressed in the documentation, e.g. should a configuration object be passed that specifies both `service` and `tags` when initializing the span?

```
tracer
  .trace('web.request', {
    service: 'my_service',
    tags: { 
       // ...
    }
  })
```

**Comment by eavestn** [comment]
I could look at the code; however, for more enterprise-y integrations, listing this will help people feel more confident in the library, itself, I believe.

**Comment by rochdev** [comment]
@eavestn You have control over the configuration options, but not the tags.

Do you think changing the statement to something like this could work?

> Each integration has its own options and can be configured individually. There is also a list of tags that get added automatically to the span created by the integration.

**Comment by eavestn** [comment]
@rochdev I would suggest something similar:

> Each integration has its own list of default tags. These tags get automatically added to the span created by the integration. Every integration is enabled by default; however, which integration is enabled can be individually configured. 

Some flavor or combination of the two. Let me know if that above makes sense based on our conversation.

**Comment by rochdev** [comment]
@eavestn I think it makes sense. I would probably just change the last part to make it clearer that there are configuration options as well.

Something like:

> however, each integration can also be enabled and configured individually.

**Comment by eavestn** [comment]
Looks good to me!

**Comment by rochdev** [comment]
Released in `v0.2.1`.
